doc/engines.rst

```diff
@@ -67,6 +67,13 @@ Stream
     This should not be confused with a CUDA stream, which corresponds to a
     Command Queue in OpenCL terminology.
 
+Stream Group
+    A group of incoming stress whose data are combined in chunks (see
```

Comment:
```suggestion
    A group of incoming streams whose data are combined in chunks (see
```

Reply:
I really need to stop using the wrong giraffes :-)